JumpscaleLibs/tools/tf_gateway/TFGateway.py

```diff
@@ -63,43 +64,45 @@ def domain_register(self, name, domain="bots.grid.tf.", record_type="a", records
         :param records: records list, defaults to None
         :type records: [type], optional is [ {"ip":machine ip}] in case of a/aaaa records
         """
+
         if not domain.endswith("."):
             domain += "."
         data = {}
         records = records or []
-        if self.redisclient.hexists(domain, name):
-            data = j.data.serializers.json.loads(self.redisclient.hget(domain, name))
+        if self.redisclient.hexists(prefix + domain, name):
```

Comment:
please make sure to install pre-commit and apply linting tools (no spaces in function calls except for `, `